### PR TITLE
Fix #6212: CWTS options in webview need to be reflected in native settings

### LINKED ISSUE #6212: [FxA Upgrade] Sync options which were deselected during Create Account flow are still showing as switched on in Firefox Account Sync settings
**Steps to reproduce**
1. Install Firefox Beta (v23.1)
2. Open app in Welcome to Firefox flow choose to Sign up.  
3. In the Create Account flow enter a valid email, password and age for the registration form and tap to Continue. 
4. Reach CWTS screen a deselect between 1 to 3 of the sync preferences. 
5. Finish up account verification and then reach iOS FF app home screen. 
6. Open menu and go to Settings> Firefox Account and view Sync Settings. 
 
**Expected result:** 
Sync options that are deselected during account creation are switched off. 

**Actual results:**
All Sync options are switched on. 

**Device information:**
Device: iPhone X(13.2.2)

Notes: 
This also occurs when creating an account from Synced Tabs menu. 

CWTS options are not saved in browser after account creation

### PR DESCRIPTION
**Pull request by garvankeeley**
The setting for this is shown here https://github.com/mozilla-mobile/firefox-ios/blob/1c4b280931e3b2aedca79fb5e02ff43144700d2e/Client/Frontend/Settings/SyncContentSettingsViewController.swift#L154

### CODE REVIEW

**Comment by eoger** [comment]
The CWTS choices would be handled by https://github.com/mozilla-mobile/firefox-ios/blob/f6d08f12953e42ec784167406e8f8bb3bfc681f8/Providers/Profile.swift#L1163 which is called when syncing.

I would suggest instead to run a sync ASAP after logging-in.

File: RustFxA/FxAWebView.swift [hunk 1]
```diff
@@ -189,6 +197,8 @@ extension FxAWebView: WKScriptMessageHandler {
 
         let auth = FxaAuthData(code: code, state: state, actionQueryParam: "signin")
         RustFirefoxAccounts.shared.accountManager.finishAuthentication(authData: auth) { _ in
+            self.profile.syncManager.onAddedAccount()
```
Comment (garvankeeley) [inline RustFxA/FxAWebView.swift]: @eoger I added this to do a sync ASAP. (the old code was calling this too)

Comment (garvankeeley) [reply RustFxA/FxAWebView.swift]: forgot to mention, that func does `self.syncEverything(why: .didLogin)`

Comment (eoger) [reply RustFxA/FxAWebView.swift]: Awesome, then we can delete the `for` loop above which might create nasty side-effects

Comment (garvankeeley) [reply RustFxA/FxAWebView.swift]: IIUC, the for-loop sets the UI prefs and is needed, the sync does not set the UI prefs in this case.

Comment (garvankeeley) [reply RustFxA/FxAWebView.swift]: Based on slack convo, removed for-loop, code works ok

**Pull request merged**